.github/workflows/unit-tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     needs: [setup-env]
 
     steps:
-      - name: Checkout under $GITHUB_WORKSPACE
+      - name: Checkout repository
```

Comment:
The variable in the old version wasn't doing anything. It just showed up in the logs as the variable, and not the value you would expect. I don't think the variable is useful anyway, if we were to make this work as was originally expected.